src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -773,13 +773,13 @@ private static JsonRpcMessage CreateError(JToken id, Exception exception)
                 foreach (var targetMap in this.targetRequestMethodToClrMethodMap)
                 {
                     targetMethod = new TargetMethod(request, targetMap.Item1, jsonSerializer, targetMap.Item2);
-                    if (targetMethod.IsFound)
+                    if (targetMethod != null && targetMethod.IsFound)
                     {
                         break;
                     }
                 }
 
-                if (!targetMethod.IsFound)
+                if (targetMethod == null || !targetMethod.IsFound)
```

Comment:
doh! I guess we already had such a test. I'll remove the extra and merge your PR. Thanks!